marketplace/cloudinary-dialog/src/index.css

```diff
@@ -0,0 +1,52 @@
+html,
+body,
+div {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: hidden;
+}
+iframe {
+  height: 100vh;
+  position: relative;
+}
+
+.logo {
+  background-image: url('https://cloudinary-res.cloudinary.com/image/upload/fl_attachment/v1/logo/for_white_bg/cloudinary_icon_for_white_bg.svg');
```

Comment:
I think it worth to copy this svg file and commit to the repo to avoid unnecessary network request. What do you think?

Reply:
fair point. This will be resolved in next commit